components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -232,7 +244,6 @@ def get_clp_connection_params_and_type(self, disable_localhost_socket_connection
 
     def dump_to_primitive_dict(self):
         d = self.model_dump(exclude={"username", "password"})
```

Comment:
in a separate PR, we shall investigate
specifying private attributes like `username` as https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.PrivateAttr
and
specifying secrets like `password` as https://docs.pydantic.dev/2.2/usage/types/secrets/

so we can completely get rid of such `dump_to_primitive_dict()` methods.

@coderabbitai create an issue to track